spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/lock/DefaultLockRepository.java

```diff
@@ -392,7 +392,7 @@ public void close() {
 	@Override
 	public boolean delete(String lock) {
 		return this.defaultTransactionTemplate.execute(
-				transactionStatus -> this.template.update(this.deleteQuery, this.region, lock, this.id)) > 0;
+				transactionStatus -> this.template.update(this.deleteQuery, this.region, lock, this.id)) == 1;
```

Comment:
I am not sure what difference with existing style.
How whatever you have done here helps with readability?

Reply:
I think reads can easily understand that only one lock ownership is expected to be deleted.

Reply:
That's debatable. Because you see: for me this does nothing.
However to save both our time discussing this I'll go ahead and merge.